container_env_files/cgw.env

```diff
@@ -1,5 +1,5 @@
 # More detailed description of the available values can be found:
-# https://github.com/gnosis/safe-client-gateway/blob/main/.env.sample
+# https://github.com/gnosis/safe-client-gateway/blob/main/.env.example
```

Comment:
the original was correct

Reply:
Ok, I did it to align with `safe-react` repo